apps/appgen/template_main.c_source

```diff
@@ -0,0 +1,38 @@
+/****************************************************************************
+ *
+ * Copyright 2018 Samsung Electronics All Rights Reserved.
```

Comment:
This is an awesome tool for TizenRT newbies.
But, please double check whether the year '2018' is reasonable in case that some app developers will create their own apps in 2019.

Reply:
You're right. It's improved. The current year will be applied when generating.